### ISSUE TITLE
Failing to render all resources - Travis build ...

### ISSUE THREAD

**Issue by door43app** [unfoldingWord-dev/ts-android#2099]
Notes
======
Failing to render all resources - Travis build 1385

Environment
======
Environment Key | Value
:----: | :----:
version | 11.0
build | 173
UDID | a4b66484d349dee1
Android Release | 5.0.2
Android SDK | 21
Brand | lge
Device | e9wifi
Log history
======
```java
4/6/17 6:13 PM  I/com.door43.translationstudio.services.BackupService: Finished backup check.
4/6/17 6:12 PM  I/com.door43.translationstudio.services.BackupService: Checking for changes
4/6/17 6:12 PM  I/com.door43.translationstudio.services.BackupService: Finished backup check.
4/6/17 6:12 PM  I/com.door43.translationstudio.services.BackupService: Checking for changes
4/6/17 6:10 PM  I/com.door43.translationstudio.services.BackupService: Finished backup check.
4/6/17 6:10 PM  I/ViewModeFragment: doScrollToPosition: position=28, offset=0
4/6/17 6:10 PM  I/ViewModeFragment: doScrollToPosition: position=0, offset=0
4/6/17 6:10 PM  I/UpdateAppTask: Migrating: /storage/emulated/0/translationStudio/translations/en_php_text_reg
4/6/17 6:10 PM  I/UpdateAppTask: Migrating: /storage/emulated/0/translationStudio/translations/en-x-demo1_phm_text_reg
4/6/17 6:10 PM  I/UpdateAppTask: Migrating: /storage/emulated/0/translationStudio/translations/en-x-demo1_php_text_ulb
4/6/17 6:10 PM  I/UpdateAppTask: Migrating: /storage/emulated/0/translationStudio/translations/en-x-demo1_luk_text_reg
4/6/17 6:10 PM  I/UpdateAppTask: Migrating: /storage/emulated/0/translationStudio/translations/en-x-demo1_1ch_text_reg
4/6/17 6:10 PM  I/UpdateAppTask: Migrating: /storage/emulated/0/translationStudio/translations/en-x-demo1_num_text_reg
4/6/17 6:10 PM  I/UpdateAppTask: Migrating: /storage/emulated/0/translationStudio/translations/en-x-demo1_php_text_reg
4/6/17 6:10 PM  I/UpdateAppTask: Migrating: /storage/emulated/0/translationStudio/translations/en-x-demo2_php_text_reg
4/6/17 6:10 PM  I/UpdateAppTask: Migrating: /storage/emulated/0/translationStudio/translations/en-x-demo1_psa_text_reg
4/6/17 6:09 PM  I/com.door43.translationstudio.services.BackupService: Checking for changes
     

```

**Comment by cckozie** [comment]
So this is an interesting one. I think there are two issues related to what I'm seeing.
I have a php project that uses the Bengali langauge (bn) as the source.
1. tS is displaying Words (in English) on the resource cards even though there is no bn Words resource.
2. If the project opens in landscape orientation with multiple chunks showing on the screen, the bottom resource chunk doesn't get rendered. The wheel just spins forever.
3. If I add the English ULB as a source but have Bengali as the selected one, then the Words do get rendered in all chunks.
4. Even if Bengali is the only source, the render problem does not occur in portrait orientation.

**Comment by cckozie** [comment]
My assumption is that Words should not get rendered unless there is a words RC for the active source language since that is the way it operates in desktop.

**Comment by neutrinog** [comment]
@cckozie originally (years ago now) it was decided to fall back to English if resources were not available in the gateway language.

**Comment by cckozie** [comment]
@neutrinog Very interesting. I tested on my Lenovo and saw the same failure. I then checked to see which app package I had downloaded/installed and it was 1384. When I upgraded to 1385, the problem went away.
So the next activity was to download the newest again to my LG. I checked the checksums on the new download vs the last and they were the same, but when I installed the new one I could not replicate the problem. User error???? Failed to actually do the install??? Don't know, but am closing this now. I'll open another issue if necessary for displaying English Words with sources other than English ULB
